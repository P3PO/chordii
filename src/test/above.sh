#!/bin/sh
echo a2crd - checking conversion of chords above lyrics...
./a2crd test/data/above.input > above.actual
diff above.actual test/data/above.expected
exitcode=$?
rm above.actual
exit $exitcode
