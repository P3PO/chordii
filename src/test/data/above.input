C    G     F
Everybody, let's have a wonderful time
C       G       F
Take it slowly, while you're feelin' fine
C          E7            F
One strage day shouldn't take you by surprise
C              G        F
But all right, hey hey, hey hey

C    Am   G
Ahhh ahhh ahhh
C    Am   G
Ahhh ahhh ahhh

F         C
Wonderful time
