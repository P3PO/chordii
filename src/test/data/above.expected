[C]Every[G]body, [F]let's have a wonderful time
[C]Take it [G]slowly, [F]while you're feelin' fine
[C]One strage [E7]day shouldn't [F]take you by surprise
[C]But all right, [G]hey hey, [F]hey hey

[C]Ahhh [Am]ahhh [G]ahhh
[C]Ahhh [Am]ahhh [G]ahhh

[F]Wonderful [C]time
